arangod/Cluster/v8-cluster.cpp

```diff
@@ -1422,8 +1423,8 @@ static void PrepareClusterCommRequest(v8::FunctionCallbackInfo<v8::Value> const&
       initTimeout = TRI_ObjectToDouble(isolate, opt->Get(context, InitTimeoutKey).FromMaybe(v8::Handle<v8::Value>()));
     }
   }
-  if (coordTransactionID == 0) {
-    coordTransactionID = TRI_NewTickServer();
+  if (coordTransactionID.empty()) {
+    coordTransactionID = TransactionId::createSingleServer();
```

Comment:
Using transaction ids here is really misleading. Especially the line
`coordTransactionID = TransactionId::createSingleServer()` takes a while to digest...
Can we convert this to a simple `uint64_t` here, so we don't confuse it with proper transactions?